src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -302,36 +293,42 @@ interface IOptionSettlementEngine {
     function option(uint256 tokenId) external view returns (Option memory optionInfo);
 
     /**
-     * @notice Returns Claim struct details about a given tokenId if that token is a
+     * @notice Returns OptionLotClaim struct details about a given tokenId if that token is a
      * claim NFT.
      * @param tokenId The id of the claim.
      * @return claimInfo The Claim struct for tokenId.
      */
-    function claim(uint256 tokenId) external view returns (Claim memory claimInfo);
+    function claim(uint256 tokenId) external view returns (OptionLotClaim memory claimInfo);
 
     /**
-     * @notice Returns the total amount of options written and exercised for all claims /
-     * option lots created on the supplied index.
+     * @notice Returns the total amount of options written and exercised for all
+     * option lot claims created on the supplied index.
      * @param optionId The id of the option for the claim buckets.
      * @param dayBucket The index of the claimBucket to return.
```

Comment:
I think dayBucket is unclear as to what the index is (my bad). can we think of something better?

Reply:
Thinking about it.

Reply:
claimBucketIndex ?

Reply:
that works!

Reply:
This is actually on the function we are aiming to remove, so leaving for now and just letting this thing build.